package.json

```diff
@@ -27,6 +27,7 @@
   "homepage": "https://github.com/UKHomeOffice/asl-schema#readme",
   "dependencies": {
     "@asl/constants": "^0.7.1",
+    "crypto": "^1.0.1",
```

Comment:
`crypto` is part of node core. You don't need this.

Reply:
derp